src/Services/API/Project/ProjectService.API/Controllers/ProjectController.cs

```diff
@@ -0,0 +1,121 @@
+﻿using ProjectService.Contract.DataTransfer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using EnterpriseAssistant.DataAccess;
+using Microsoft.EntityFrameworkCore;
+using EnterpriseAssistant.DataAccess.Entities;
+using Mapster;
+
+namespace ProjectService.API.Controllers;
+
+[AllowAnonymous]
+[ApiController]
+[Route("api/project")]
+public class ProjectController : ControllerBase
+{
+    private readonly EnterpriseAssistantDbContext _context;
+
+    public ProjectController(EnterpriseAssistantDbContext context)
+    {
+        _context = context;
+    }
+
+    /*
+    [HttpPost]
+    [SwaggerOperation(Summary ="Create project", Description = "Create a project")]
+
+    public async Task<ActionResult<ProjectCreateDto>> CreateProject([FromBody] ProjectCreateDto project)
+    {
+        var result = await _mediator.Send(new CreateProject(project));
+        return Ok(result);
+    }
+
+
+    [HttpGet("{projectId:long}")]
+    [SwaggerOperation(Summary = "Get prject by Id", Description = "Get project")]
+    public async Task<ActionResult<IEnumerable<ProjectDto>>> GetProjectById(
+        [Range(1, long.MaxValue), FromRoute] long projectId)
+    {
+        var result = await _mediator.Send(new GetProjectById(projectId));
+
+        return result.Match<ActionResult>(Ok, x => NotFound());
+    }
+
+    [HttpDelete("{projectId:long}")]
+     [SwaggerOperation(Summary = "Delete project by id", Description = "delete project")]
+      public async Task<IActionResult> ProjectDelete(long id)
+     {
+
+     }
+    */
+
+    [HttpPost]
+    [SwaggerOperation(Summary = "Add project", Description = "add project")]
+    public async Task<ActionResult<Project>> PostProject(ProjectCreateDto request)
```

Comment:
I suggest using CreateProject or just Create instead of "Post", this name will describe method purpose more clearly